### DISCUSSION TITLE
Copilot "Failed to get copilot token" OR "GitHub Copilot could not connect to server"

### DISCUSSION THREAD

**Discussion by saver711** [community/community#16211] [Copilot Conversations]
Anyone getting these errors >>
I was having these errors and I solved them by correcting the time zone on my pc.

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Comment by PythonLimited** [comment]
I am/was getting the same issue.
Over the past days the extension also required me to re-login every time I opened a vscode instance.
It ultimately resulted in the error you described above -> a reinstall os vscode fixed it?

Edit:
still requires me to "Sign in to GitHub" every time for some reason...

**Reply by saver711** [reply]
try correcting the time zone on your PC.
